src/scripts/components/foundations/svgs/FilesUploaderSvg.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+
+type Props = {}
+
+const SVG = styled.svg`
+  display: block;
+  height: 100%;
+  width: 100%;
+`
+
+class FilesUploaderSvg extends Component<Props, void> {
```

Comment:
having a components for just a static svg it seems quite too much.

Reply:
I think we should come up w/ a universal svg solution. This may be a fine way to go, we can even make this into a stateless functional component so it is cleaner. But I think we definitely can't revert back to the `Svgs.js` component that this PR deletes. That component would render all of our app SVGs every time we use one svg, which is not scalable. This method to me is preferable.

Reply:
after reading this https://css-tricks.com/creating-svg-icon-system-react/ i agree with you guys.
we could pass props to svg and make awesome stuff.